.github/workflows/ci.yml

```
@@ -476,11 +493,54 @@ jobs:
+
+    - name: Configure Git
+      run: |
+        git config --global user.name "github-actions[bot]"
+        git config --global user.email "github-actions[bot]@users.noreply.github.com"
+
+    - name: Merge main into develop
+      run: |
+        echo "🔄 Merging main branch changes back to develop..."
+        git fetch origin main
+        git merge origin/main --no-edit
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)